### PR TITLE
Enable Qt imageformats/iconengines/accessible plugins when building with static Qt

### PR DESCRIPTION
**Pull request by rryan**

### CODE REVIEW

**Comment by sblaisot** [comment]
not better :(

try with 
`QTPLUGIN += qsvg qico`

and 

`
#include <QtPlugin>
Q_IMPORT_PLUGIN(qsvg)
Q_IMPORT_PLUGIN(qico)
`

http://www.formortals.com/how-to-statically-link-qt-4/

**Comment by rryan** [comment]
> http://www.formortals.com/how-to-statically-link-qt-4/

Nice -- done. (We don't use qmake so QTPLUGIN += ... isn't applicable. I think that does essentially what we've done in build/depends.py).

File: src/mixxxapplication.cpp [hunk 1]
```diff
@@ -1,10 +1,24 @@
 #include <QtDebug>
 #include <QTouchEvent>
-
 #include "mixxxapplication.h"
 #include "control/controlproxy.h"
 #include "mixxx.h"
 
+// When linking Qt statically on Windows we have to Q_IMPORT_PLUGIN all the
+// plugins we link in build/depends.py.
+#ifdef QT_NODLL
```
Comment (sblaisot) [inline src/mixxxapplication.cpp]: don't we need to enforce the "on windows" part of your comment here ?
How are imageformats plugins under Linux/MacOS ?

**Comment by sblaisot** [comment]
\o/ this works !

File: src/mixxxapplication.cpp (hunk 1 above)
Comment (rryan) [inline src/mixxxapplication.cpp]: QT_NODLL is a windows-only define, so I thought that was enough.

Comment (sblaisot) [inline src/mixxxapplication.cpp]: makes sense. thx.

**Comment by sblaisot** [comment]
LGTM

**Pull request merged**